src/assets/index.js

```diff
@@ -116,7 +116,7 @@ const analyze_click = () => {
     document.getElementById("leaf_input").value = "";
     document.getElementById("showImage").src = DUMMY_URL;
     uploadButtonSpan.innerHTML = "Upload a file";
-    document.getElementById("img-lab").innerHTML = "";
```

Comment:
Hey, Can you check and update the behavior of img-label when analyze other crop is clicked from result page!
I think only a few changes are needed here! create a new PR and refer this, you will not get any points for new one :sweat_smile:  !

Reply:
Okay. Yeah sure, didn't consider that case.